backend/open_webui/utils/embeddings.py

```
@@ -0,0 +1,90 @@
+        }
+    else:
+        models = request.app.state.MODELS
+
+    model_id = form_data.get("model")
+    if model_id not in models:
+        raise Exception("Model not found")
+    model = models[model_id]
+
+    # Access filtering
+    if not getattr(request.state, "direct", False):
+        if not bypass_filter and user.role == "user":
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)